app/controllers/api/messaging.php

```diff
@@ -0,0 +1,2145 @@
+    ->label('sdk.description', '/docs/references/messaging/delete-subscriber.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_NOCONTENT)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_NONE)
+    ->param('topicId', '', new UID(), 'Topic ID.')
+    ->param('subscriberId', '', new UID(), 'Subscriber ID.')
+    ->inject('dbForProject')
+    ->inject('response')
+    ->action(function (string $topicId, string $subscriberId, Database $dbForProject, Response $response) {
+        $topic = Authorization::skip(fn () => $dbForProject->getDocument('topics', $topicId));
+
+        if ($topic->isEmpty()) {
+            throw new Exception(Exception::TOPIC_NOT_FOUND);
+        }
+
+        $subscriber = $dbForProject->getDocument('subscribers', $subscriberId);
+
+        if ($subscriber->isEmpty() || $subscriber->getAttribute('topicId') !== $topicId) {
+            throw new Exception(Exception::SUBSCRIBER_NOT_FOUND);
+        }
+
+        $subscriber = $dbForProject->deleteDocument('subscribers', $subscriberId);
+        Authorization::skip(fn () => $dbForProject->decreaseDocumentAttribute('topics', $topicId, 'total', 1));
+
+        $response
+            ->setStatusCode(Response::STATUS_CODE_NOCONTENT)
+            ->noContent();
+    });
+
+App::post('/v1/messaging/messages/email')
+    ->desc('Create an email.')
+    ->groups(['api', 'messaging'])
+    ->label('audits.event', 'message.create')
+    ->label('audits.resource', 'message/{response.$id}')
+    ->label('scope', 'messages.write')
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN, APP_AUTH_TYPE_KEY])
+    ->label('sdk.namespace', 'messaging')
+    ->label('sdk.method', 'createEmail')
+    ->label('sdk.description', '/docs/references/messaging/create-email.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_CREATED)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_MESSAGE)
+    ->param('messageId', '', new CustomId(), 'Message ID. Choose a custom ID or generate a random ID with `ID.unique()`. Valid chars are a-z, A-Z, 0-9, period, hyphen, and underscore. Can\'t start with a special char. Max length is 36 chars.')
+    ->param('to', [], new ArrayList(new Text(Database::LENGTH_KEY), 1), 'List of Topic IDs or List of User IDs or List of Target IDs.')
+    ->param('subject', '', new Text(998), 'Email Subject.')
+    ->param('content', '', new Text(64230), 'Email Content.')
+    ->param('description', '', new Text(256), 'Description for message.', true)
+    ->param('status', 'processing', new WhiteList(['draft', 'processing']), 'Message Status. Value must be either draft or processing.', true)
+    ->param('html', false, new Boolean(), 'Is content of type HTML', true)
+    ->param('deliveryTime', null, new DatetimeValidator(requireDateInFuture: true), 'Delivery time for message in ISO 8601 format. DateTime value must be in future.', true)
```

Comment:
Not sure about the naming convention. We use somethingAt in many places, what do we think about that?

Reply:
@eldadfux , we have two attributes, deliveredAt (signifies when the message was delivered to the user) and deliveryTime( time when message is scheduled at), we can rename this to scheduleAt, how does that sound?

Reply:
@fanatic75 Renaming to scheduleAt sounds good :+1: